test/controllers/welcome_controller_test.rb

```diff
@@ -2,6 +2,10 @@
 
 class WelcomeControllerTest < ActionDispatch::IntegrationTest
   test "should get index" do
+    # TODO: what is this welcome_index_url? This text only appears in this one spot in the project
```

Comment:
Yep, `welcome_index_url` is a Rails-generated URL for `GET welcome/index`

You can see all of the url helpers by going to a terminal prompt and entering `rails routes`. (Use a very wide terminal window to see the output clearly.)

Reply:
Thanks